modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module ClaimsApi
+  class PoaFormBuilderJob
+    include Sidekiq::Worker
+
+    def perform(power_of_attorney_id)
+      power_of_attorney = ClaimsApi::PowerOfAttorney.find power_of_attorney_id
+      signed_pdf = power_of_attorney.sign_pdf
+      pdf_constructor = ClaimsApi::PowerOfAttorneyPdfConstructor.new(power_of_attorney.id)
+      page1 = pdf_constructor.fill_pdf(signed_pdf[:page1], 1)
+      page2 = pdf_constructor.fill_pdf(signed_pdf[:page2], 2)
+      output_path = "/tmp/#{power_of_attorney_id}_final.pdf"
```

Comment:
Does this need some random file path goodness, or is it safe to use only the id?

Reply:
The ID should be fine so we dont have to persist a separate path somewhere between generate and upload